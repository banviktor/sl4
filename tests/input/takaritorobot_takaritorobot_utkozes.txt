randomization off
questions off
start_game 2
5 5
2 1
toggle_glue
inputvector 0 5
jump
jump
I
5 4.05
jump
jump
I
5.95 5
0 5
list_cleaner_robots
exit
