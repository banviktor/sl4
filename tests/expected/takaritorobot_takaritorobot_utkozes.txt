Véletlenszerű események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Ragacs be.
Olajfolt ki.
Inputvector: 0.0, 1.0
Ragacsfolt lehelyezve.
Új pozíció: 5.0, 6.0
Ez a(z) 1. kör.
A Sárga Robot következik.
Új pozíció: 2.0, 1.0
Takarítórobot indul útnak innen: 5.0, 4.05
Ez a(z) 2. kör.
A Piros Robot következik.
Új pozíció: 5.0, 7.0
Ez a(z) 2. kör.
A Sárga Robot következik.
Új pozíció: 2.0, 1.0
Takarítórobot indul útnak innen: 5.95, 5.0
Takarítórobotok listája:
	Takarítórobot: 5.0, 5.05
	Takarítórobot: 4.95, 6.0
